pytket/extensions/qiskit/backends/ibm.py

```diff
@@ -128,7 +114,7 @@ def __init__(self) -> None:
 
 
 class IBMQBackend(Backend):
-    _supports_shots = True
+    _supports_shots = False
```

Comment:
This comes from #29 . (By the way, this has already been reviewed and merged to `develop`, this PR is just for the merge to `main`, but good to have another pair of eyes on it!)

This is because the qiskit `Sampler` runtime does not return a shots table, we only get counts.

Reply:
Oh oops. Well thanks for replying anyway!